packages/chakra-ui/src/Widgets/Widgets.tsx

```diff
@@ -0,0 +1,21 @@
+import BaseInput from '../BaseInput/BaseInput'
+import CheckboxesWidget from '../CheckboxesWidget/CheckboxesWidget'
+import CheckboxWidget from '../CheckboxWidget/CheckboxWidget'
+import RadioWidget from '../RadioWidget/RadioWidget'
+import RangeWidget from '../RangeWidget/RangeWidget'
+import SelectWidget from '../SelectWidget/SelectWidget'
+import TextareaWidget from '../TextareaWidget/TextareaWidget'
+import UpDownWidget from '../UpDownWidget/UpDownWidget'
+
+const widgets = {
+  BaseInput,
+  CheckboxWidget,
+  CheckboxesWidget,
+  radio: RadioWidget,
+  RangeWidget,
+  select: SelectWidget,
+  TextareaWidget,
+  UpDownWidget
+}
```

Comment:
For the "Widgets" tab -- setting `ui:widget` to `hidden` should completely hide the item, but in this theme it's showing up:

![image](https://user-images.githubusercontent.com/1689183/111927236-6b935300-8a86-11eb-9693-7b72d2d2fff9.png)